### DISCUSSION TITLE
c# usage

### DISCUSSION THREAD

**Discussion by davilacervantes** [tshino/softcam#41] [General]
hello do you have some example to use this vcam from c#? or some guidance,  it is possible?

thanks!

**Comment by tshino** [comment]
Thank you for asking!
For now, I have no example code for C#.
But I believe it is possible to make C# binding of Softcam as I already made Python binding (#13).

**Reply by davilacervantes** [reply]
thanks Tshino, 

another behavior i notice, if the softcam is registered the real webcamera cannot be used in chrome or edge it just not recognize until the softcam is unregistered

this is a problem that can be fixed?

**Reply by tshino** [reply]
As far as I have experienced, we can always select which camera to use including virtual cameras through the settings on the browser.
If your real webcam have disappeared or been disabled on the settings when the softcam exists, it's unexpected behavior for me.
